### ISSUE TITLE
Table 4 PointNet++ results

### ISSUE THREAD

**Issue by sheshappanavar** [hkust-vgd/scanobjectnn#22]
I tried to reproduce the results and got following for PointNet++ (Table 4 Row 4 PointNet++)

![image](https://user-images.githubusercontent.com/2320460/91647870-2f3acf80-ea2e-11ea-82f3-e3a1bb0eb32a.png)

I am surprised in each category I have higher accuracy. Please let me know if I did any mistakes while training.
After downloading and unzipping h5_files.zip I have used the following files for training and testing

OBJ only
---------
main_split_nobg/training_objectdataset.h5
main_split_nobg/test_objectdataset.h5

OBJ bg
-------
main_split/training_objectdataset.h5
main_split/test_objectdataset.h5

PB_T25
--------
main_split/training_objectdataset_augmented25_norot.h5
main_split/test_objectdataset_augmented25_norot.h5

PB_T25_R
----------
main_split/training_objectdataset_augmented25rot.h5
main_split/training_objectdataset_augmented25rot.h5

PB_T50_R
----------
main_split/training_objectdataset_augmentedrot.h5
main_split/test_objectdataset_augmentedrot.h5

PB_T50_RS
------------
main_split/training_objectdataset_augmentedrot_scale75.h5
main_split/test_objectdataset_augmentedrot_scale75.h5

Thanks in Advance

**Comment by mikacuy** [comment]
Hi,

The difference might just be a result of not having a random seed in the train script. So the last checkpoint is not exactly the same. I wasn't able to set a seed when I was creating the result. I apologize for this.

**Comment by mikacuy** [comment]
Yes, that's right I didn't use a random seed in the training script. I apologize if that caused confusions.

Best,
Mika
